src/_provisional.scss

```diff
@@ -149,6 +149,528 @@
+$gcweb-input-height-base: ($gcweb-line-height-computed + ($gcweb-padding-base-vertical * 2) + 2);
+$gcweb-input-height-small: (floor($gcweb-font-size-small * $gcweb-line-height-small) + ($gcweb-padding-small-vertical * 2) + 2);
+
+$gcweb-form-group-margin-bottom: 25px;
+
+
+/*NOTE:
+also gotta keep WET overrides in mind for things like undoing hardocoded heights for certain elements
+
+
+
+Outcome of chat with Pierre:
+-eliminate gc-form-font-new class (and scrap namesake placeholder selector since it'll now only be used once)... won't be able to do that without breaking the Bootstrap docs test page (many examples of things affected by GCWeb's small font size overrides aren't inside <form> elements and can't be overidden any other way unless I use a top-level <form>)
+-my call for text-shadow in buttons (make it all or nothing on all similar buttons or call-to-action-only)
+-make call to action buttons as much like normal buttons as possible -except the margins (call to actions font size WILL decrease to match normal buttons)
+-don't give people a way of overriding the call to action margins in provisional CSS? (can't remember what we decided for this... but I think we decided against it)
+-add an example page(s) to allow people to compare before/after (ripped off from my current test pages)
+-add upgrade step notes in the CSS and/or PR (like what to do to promote my variables from provisional to final... basically make it straightforward for PP or me to finalize the selectors in 6+ months)
+
+
+NOTES:
+-Why is there a noticeable height difference between dropdowns and text inputs? Seems to be related to line-height: normal; on select and a real line height on inputs
+-That silly arrow placement in "Buttons with dropdowns" is beyond my control (was an intentional WET override with the understanding that would break)
+-Borders in basic btn-group examples don't look as dark as justified ones (WET+GCWeb)... caused by a WET override (https://github.com/wet-boew/wet-boew/pull/5065)
+-There's weird inexplicable spacing in this GCWeb example (doesn't exist in WET... I think it's a GCWeb quirk because it happens even without the giant form font class):
+<div class="btn-group btn-group-xs" role="group" aria-label="...">...</div>
+-Seems to be compatible with "gc-chckbxrdio" (#1682), but some of the latter's font-size and line-height properties deviate from the new font variables... tried changing a few to use variables of similar sizes, kept line-height: 2 as-is
+
+TODOS:
+-IIRC disabled inputs are looking darker than expected due to WET/GCWeb overrides
+-I'd like to maybe adjust small/large line heights to be different from GCWeb's default
+-Separate temp Bootstrap selectors from GCWeb-specific ones (e.g. call to action, 11px margin-top for radio buttons, etc...)
+
+*/
+
+
+%gcweb-btn {
+	@include button-size($gcweb-padding-base-vertical, $gcweb-padding-base-horizontal, $gcweb-font-size-base, $gcweb-line-height-base, $gcweb-btn-border-radius-base);
```

Comment:
New default (or "md" if you wish) button padding should match the [current CTA pattern](https://wet-boew.github.io/GCWeb/templates/content-en.html#call-to-action). However, I noticed that .54em was better looking than .58em when zooming in on a button used in pair with inline text boxes, so I went with .54em for fielflow buttons.

Reply:
IIRC the button sizing changes this PR makes are meant to be based on the premise of increasing each button style's font size to more or less match one size above what they'd normally be (e.g. GCWeb's ``btn`` class would sorta resemble WET's ``btn-lg`` class, etc...).

The current call to action (CTA) button styles are inconsistent with that. It's not based on the same font sizing scheme or design principles (like margins) as Bootstrap/WET's button styles. So I can't really turn GCWeb's medium button styles into an exact match for CTA without causing a whole lot of other headaches.

For example:
* GCWeb's default font size is 20px whereas the CTA button font size is 22px.
* Generic button classes don't use outer margins, whereas CTA does.
* CTA buttons use text shadows, whereas Bootstrap/WET's buttons don't.

IIRC past discussions I've had about this indicated that I'd be able to pitch changes to the CTA button styles to match whatever generic "medium" button style comes out of this PR. With the only exception being the CTA button's margins (although I was thinking of allowing the margins to be undone if a ``provisional`` class is used on a CTA button). @duboisp and @delisma might need to chime in on this.

Btw I also have many more button size variations to contend with and need to ensure their sizing differences are easy to perceive. I'd of liked to post a screenshot of all my side-by-side button size examples but can't do so atm since I'd need to reset my GCWeb build system to do it lol.